manim/mobject/coordinate_systems.py

```diff
@@ -194,8 +194,8 @@ class Axes(VGroup, CoordinateSystem):
 
     def __init__(
         self,
-        x_range=None,
-        y_range=None,
+        x_range: Optional[Sequence[float]] = None,
+        y_range: Optional[Sequence[float]] = None,
```

Comment:
I don't `np.ndarray` comes under `Sequence`.

Reply:
Can you confirm?

Reply:
`np.ndarray` is a `Sequence` and Sequence is specific enough here.
https://docs.python.org/3/glossary.html#term-sequence

Reply:
I also agree with anton.
We also chose Sequence for all coordinates, which are np arrays:
https://manimce--1496.org.readthedocs.build/en/1496/contributing/documentation.html#adding-type-hints-to-functions-and-parameters